CONTRIBUTING.md

```diff
@@ -0,0 +1,40 @@
+# Contributing
+
+CLIMA encourages Pull Requests (PRs) and contributions.
+The easiest way to contribute is by running code and letting us know what's wrong by creating issues.
+In the case that you can specifically pinpoint the issue within the code, please consider submitting a PR with these changes.
+
+To contribute, we assume that you already have a basic understanding of git and version control as all tools are accessible on Github; however, we will discuss the general workflow for submitting code for review to CLIMA.
+
+## Forks and branches
+
+If you are already a collaborator for CLIMA, please create branches in the main CLIMA repository prefaced by your initials.
+Please also try to keep branch names significant and understandable as it relates to the changes being made.
+For example, if your name is Josh James Johnson and you are adding a PR to change Julia formatting, it might be called `jjj/julia-formatting`.
+
+If you are not a collaborator, then please create a CLIMA fork and create a feature branch from there.
+Please try your best not to create a PR from the master branch of your own fork as this can create a lot of challenging when merging or rebasing the PR later.
+
+Before submitting a PR, please make sure you are rebased to the current master by using `git rebase` on your feature branch.
+
+## Formatting
+
+For the most part, we follow the [YASGuide](https://github.com/jrevels/YASGuide) for Julia formatting, with small exceptions covered in the [Coding Conventions](https://climate-machine.github.io/CLIMA/latest/CodingConventions.html) section of the documentation.
+
+In addition to this, once you are happy with your PR, please apply [JuliaFormatter.jl](https://github.com/domluna/JuliaFormatter.jl) to all changed files in the repository.
+
+To apply our formatter settings to all changed files, run:
+```
+julia .dev/format.jl
+```
+
+Formatting changes should be done as a separate commit, ideally the last commit of the pull request (you may want to leave it until all other changes have been approved).
+
+## Bors and CI
+
+All commits that end up in the CLIMA repository must pass Continuous Integration (CI).
+When a PR is updated, it will automatically be run on Microsoft Azure for Linux, Windows, and MacOS; however, because CLIMA is heterogeneous and must also run on GPU hardware, we also manually launch CI with Bors.
+
+To test to see if all bors CI will pass, please type `bors test` in a separate comment in the PR.
```

Comment:
This should be `bors try`.

Reply:
@leios what's the status of this?

Reply:
Kiran and I were having some discussions about what to put into it and who the audience should be. I'll push a new version in ~30 minutes, just finishing up some KA stuff.